.github/workflows/ci.yml

```diff
@@ -57,9 +74,9 @@ jobs:
       fail-fast: false
       matrix:
         python:
-          - name: CPython 3.6
-            tox: py36
-            action: 3.6
+          - name: CPython 3.8
+            tox: py38
+            action: 3.8
```

Comment:
Any particular reason for this to move to 3.8 instead of 3.9?  It seems reasonable to bump it up, since CPython 3.6 is nearing the end of its supported life.  All else equal, I guess 3.9 would be a better new target since it will be supported for longer ... but maybe there are other concerns?

Reply:
I figured that a little extra stability isn't a bad thing on the checks.  They are there to work, not exercise against latest.  And sure, 3.9 isn't all that new, but it doesn't have a patch release yet and my brain works a bit better without nuance.  Just use the second-most-recent minor and done.  When adding 3.10 to the test matrix we could bump this to 3.9.  (Though sure, I don't think I actually apply this explicitly anywhere else.  Maybe next time I will.  `:]`)

I guess I could afford a brief comment as to the intent.